src/library/Button/styled.js

```diff
@@ -63,7 +62,7 @@ export const Content: StyledComponent<{ [key: string]: any }> = styled('span', {
   };
 });
 
-export const Inner: StyledComponent<{ [key: string]: any }> = styled('span')({
+export const Inner: StyledComponent<{ [key: string]: any }> = styled('span', {
```

Comment:
hmm.... there's more of this. and i think some of it was there BEFORE my merge
```
$ grep -r "\')({" src/library
src/library/Card/styled.js:export const CardTitleInner: StyledComponent<{}> = styled('div')({
src/library/Dialog/styled.js:export const DialogIEWrapper: StyledComponent<{}> = styled('div')({
src/library/Button/styled.js:export const Inner: StyledComponent<{ [key: string]: any }> = styled('span')({
src/library/Table/styled.js:}> = styled('span')({
```
moreover, this pattern is found everywhere in the website code:
```
$ grep -r "\')({" src/website
src/website/app/Canvas.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/Canvas.js:const Triangles = styled('div')({
src/website/app/pages/Color/index.js:const RampHolder = styled('div')({
src/website/app/demos/Dropdown/Dropdown/examples/controlled.js:const Root = styled('div')({
src/website/app/demos/Dropdown/Dropdown/examples/rtl.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Dropdown/Dropdown/examples/placement.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Dropdown/Dropdown/examples/customItem.js:const _DemoLayout: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Dropdown/Dropdown/examples/customMenu.js:const _DemoLayout: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Dropdown/Dropdown/examples/wide.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Tooltip/Tooltip/examples/controlled.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Tooltip/Tooltip/examples/placement.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Tooltip/Tooltip/bestPractices.js:const DemoCardLayout = styled('div')({
src/website/app/demos/Tooltip/Tooltip/bestPractices.js:const VerticalLayout = styled('div')({
src/website/app/demos/Tooltip/Tooltip/bestPractices.js:const HorizontalLayout = styled('div')({
src/website/app/demos/Popover/Popover/examples/controlled.js:const Root = styled('div')({
src/website/app/demos/Popover/Popover/examples/rtl.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Popover/Popover/examples/placement.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Popover/Popover/examples/title.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Popover/Popover/examples/customContent.js:const _DemoLayout: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Popover/common/ScrollBox.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Popover/common/DemoContent.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/common/DemoForm.js:const Root: StyledComponent<{ [key: string]: any }> = styled('form')({
src/website/app/demos/Link/Link/examples/link.js:const Big: StyledComponent<{ [key: string]: any }> = styled('p')({
src/website/app/demos/Link/Link/examples/link.js:const Bold: StyledComponent<{ [key: string]: any }> = styled('p')({
src/website/app/demos/Link/Link/examples/link.js:const Serif: StyledComponent<{ [key: string]: any }> = styled('p')({
src/website/app/demos/Link/Link/examples/link.js:const Small: StyledComponent<{ [key: string]: any }> = styled('p')({
src/website/app/demos/Menu/MenuGroup/bestPractices.js:const DemoLayout = styled('div')({
src/website/app/demos/Menu/MenuItem/bestPractices.js:const DemoLayout = styled('div')({
src/website/app/demos/Menu/common/customItem.js:    const Root = styled('div')({
src/website/app/demos/Menu/common/customItem.js:    const Work = styled('span')({
src/website/app/demos/Menu/common/customItem.js:    const Content = styled('span')({
src/website/app/demos/Menu/Menu/bestPractices.js:const Root = styled('div')({
src/website/app/demos/Menu/MenuDivider/bestPractices.js:const DemoLayout = styled('div')({
src/website/app/demos/Text/Text/examples/nested.js:      const CustomDiv = styled('div')({
src/website/app/demos/Select/common/DemoLayout.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Select/Select/examples/placement.js:const Root: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Select/Select/examples/customItem.js:const _DemoLayout: StyledComponent<{ [key: string]: any }> = styled('div')({
src/website/app/demos/Select/Select/examples/customMenu.js:const _DemoLayout: StyledComponent<{ [key: string]: any }> = styled('div')({
```
so I'm a little worried we might have a bigger problem on our hand.